swagger-templates/python/model.mustache

```diff
@@ -64,10 +64,10 @@ class {{classname}}(object):
         """
 {{#isEnum}}
         allowed_values = [{{#allowableValues}}{{#values}}"{{{this}}}"{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]
-        if {{name}} not in allowed_values:
+        if {{{name}}} not in allowed_values:
```

Comment:
It is from the newest template in swagger. I just found my first commit (original template is not from the newest one but my changes are based on the newest template) haha